Automattic/wp-calypso
# Remove verify email banner from launchpad

Related Issue: https://github.com/Automattic/wp-calypso/issues/73868
Related PR: https://github.com/Automattic/wp-calypso/pull/74135

### Proposed Changes

This PR removes the Email Verification Banner from Launchpad. We've added a Verify Email task to the Launchpad tasks for Newsletters ([see PR](https://github.com/Automattic/wp-calypso/pull/74135)), and no longer need the banner. 
  - REMOVE COMPONENT FILE. The EmailVerificationBanner is not used anywhere else, so I've completely removed the file. 
  - REMOVE TEST FILE. Tests for step-content.tsx were entirely focused on testing this banner, so I've removed that test file completely. We might consider additional tests for that file, but that can be handled in a separate PR.
 
<img width="1461" alt="email verification banner" src="https://user-images.githubusercontent.com/21228350/224802768-092a960d-02f3-4242-8dc4-1df2e6a4403a.png">
  
### Testing 

**Review Time: Short**
**Testing Time: Medium**

1. Check out this brand and run yarn and yarn start if needed. 
2. Log out of WordPress.com or use a fresh browser where you are not logged in. 
3. Set up a temporary email at https://temp-mail.org/
4. Create a new newsletter site at http://calypso.localhost:3000/setup/newsletter/intro. You'll need to create a new account. Use the email setup in Step 3. 
5. Click through to Launchpad. 
6. TEST: Confirm the email verification banner no longer shows, even when your email is not verified.
7. TEST: Smoke test other aspects of Launchpad to confirm it continues to work as expected. Consider verifying your email, and coming back to Launchpad, and again smoke test to confirm everything works as expected. 
8. TEST: Confirm Launchpad unit tests still pass by running `yarn run test-client client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/` from your local repository root.
